ES/ESmy007.xml

```diff
@@ -30,99 +30,448 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                  <num value="6">፮</num>
                                  <dim unit="leaf">8</dim>
-                                 <locus from="45r" to="52v"/> ፮VI(8/fols. 45r-52v) </item>
-                              <item xml:id="q8">
+                                 <locus from="45r" to="52v"/>
+                               </item>
+
+                              <item xml:id="q8" n="7">
+                                  <num value="7">፯</num>
                                  <dim unit="leaf">8</dim>
-                                 <locus from="53r" to="60v"/> ፯VII(8/fols. 53r-60v) </item>
-                              <item xml:id="q9">
+                                 <locus from="53r" to="60v"/>
+                               </item>
+
+                              <item xml:id="q9" n="8">
+                                  <num value="8">፰</num>
                                  <dim unit="leaf">8</dim>
-                                 <locus from="61r" to="68v"/> ፰VIII(8/fols. 61r-68v) </item>
-                              <item xml:id="q10">
+                                 <locus from="61r" to="68v"/>
+                               </item>
+
+                              <item xml:id="q10" n="9">
+                                <num value="9">፱</num>
                                  <dim unit="leaf">8</dim>
-                                 <locus from="69r" to="76v"/> ፱IX(8/fols. 69r-76v) </item>
-                              <item xml:id="q11">
+                                 <locus from="69r" to="76v"/>
+                               </item>
+
+                              <item xml:id="q11" n="10">
+                                <num value="10">፲</num>
                                  <dim unit="leaf">8</dim>
-                                 <locus from="77r" to="84v"/> ፲X(8/fols. 77r-84v) </item>
-                              <item xml:id="q12">
+                                 <locus from="77r" to="84v"/>
+                               </item>
+
+                              <item xml:id="q12" n="11">
+                                  <num value="11">፲፩</num>
                                  <dim unit="leaf">8</dim>
-                                 <locus from="85r" to="92v"/> ፲፩XI(8/fols. 85r-92v) </item>
-                              <item xml:id="q13">
+                                 <locus from="85r" to="92v"/>
+                               </item>
+
+                              <item xml:id="q13" n="12">
+                                  <num value="12">፲፪</num>
                                  <dim unit="leaf">8</dim>
-                                 <locus from="93r" to="100v"/> ፲፪XII(8/s. l.: 3, 6, stubs after 2,
-                                 5/fols. 93r-100v) </item>
-                              <item xml:id="q14">
+                                 <locus from="93r" to="100v"/>
+                                 s.l.: 3, stub after 2
```

Comment:
change to
`s.l.: 3, 6, stubs after 2, 5`
(does not calculate otherwise)

Reply:
Corrected.